build/build.js

```diff
@@ -22,8 +23,16 @@ const get = async (url, binary = true) => {
   return binary ? res.buffer() : res.text();
 };
 
+const force = process.argv.slice(2).some((arg) => ['--force', '-f'].includes(arg)) || process.env.FORCE !== 'true';
```

Comment:
```suggestion
const force = process.argv.slice(2).some((arg) => ['--force', '-f'].includes(arg)) || process.env.FORCE === 'true';
```

Reply:
opsie

Reply:
eslint won't let me do this way unfortunately